README.md

```diff
@@ -64,6 +64,12 @@ Here's a sample configuration with using `use-package`:
               (("C-c n i" . org-roam-insert))))
 ```
 
+`org-roam-graph` by default expects to find the `dot` executable 
+from the `graphviz` package in the `$PATH`.
+Ensure `graphviz` is installed if you want to use this feature
+or customize your configuration for `org-roam-graph` to use a 
+different tool.
```

Comment:
This section is not entirely correct. It doesn't have to be in env variable `$PATH`, but it needs to be in Emacs-lisp list `exec-path`. Ensuring graphviz is installed doesn't make the path to the executable available to Emacs either, that requires a bit more careful customization, but is typically resolved with package `exec-path-with-shell`.

Reply:
Thanks for the suggestion. I changed it a bit.